integration/src/main/java/scripts/datageneration/regenerate_perf_data.sql

```diff
@@ -0,0 +1,47 @@
+-- This function inserts a semi-random linked account and passport with
+-- the given user_id and external_user_id
+CREATE PROCEDURE insert_test_record(user_id text, external_user_id text)
+LANGUAGE SQL
+AS $$
```

Comment:
Can you explain the usage of the $$s? I haven't seen the double bling before (and it's hard to google)

Reply:
Here's some documentation on it: https://www.geeksforgeeks.org/postgresql-dollar-quoted-string-constants/

I think it's only really used for functions and stored procedures though (to enclose blocks of code that will be reused).